README.md

```diff
@@ -1,76 +1,231 @@
+        <li>From the version page you can choose options, launch the version, and more!</li>
+</div>
+<br>
+
+<!--MANAGE VERSIONS-->
+<div>
+    <h2><b>Manage Versions</b></h2>
+    <span>
+        <a href="../../">BSManager</a> allows you to manage your versions of Beat Saber. It gives you the ability to download any version of Beat Saber as long as you have a Steam account that already owns the game. Using <a href="../../">BSManager</a>, you can easily manage and update your collection of Beat Saber versions.
+    </span>
+</div>
+<br>
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213008824-25cfdda3-6350-42dc-aa53-2132b4b8e838.mov
+</div>
+
+<!--CUSTOM UR INSTANCES-->
+<div>
+    <h2><b>Customize your instances</b></h2>
+    <span>
+        The tool will allow you to completely customize each of your downloadable versions, you can change the color of the icon, choose a name for the instance and finally install your mods and map according to the Beat Saber version! 
+    </span>
+</div>
+<br>
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213296794-74a1e20b-12dc-44ec-a67f-e25d7010bd3d.mov
+</div>
+
+<!--INSTALL MODS-->
+<div>
+    <h2><b>Install Mods</b></h2>
+    <span>
+        <a href="../../">BSManager</a> allows you to manage Beat Saber mods in a quick and easy way. It uses the BeatMod API to allow you to install, update or uninstall mods for each version of your collection. So the tool can help you manage them efficiently.
+    </span>
+</div>
+<br>
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213296816-bdaafc90-da76-4835-8d83-9ac6a4e1c549.mov
+</div>
+
+<!--INSTALL MAPS-->
+<div>
+    <h2><b>Install Maps</b></h2>
+    <span>
+        In addition to managing mods, the tool that allows you to manage and organize custom beatmaps existing on <a href="https://beatsaver.com">Beat Saver</a>, a popular site for sharing custom beatmaps for beat Saber. The tool allows you to view and organize your beatmaps, as well as link different versions of the game to share all your maps together. 
+    </span>
+</div>
+<br>
+<div align="center">
+
+https://user-images.githubusercontent.com/40648115/213296838-f775fe83-866d-4764-a7d0-648dbb8ec983.mov
+</div>
+
+<!--ONE CLICK-->
+<div>
+    <h2><b>One Click</b></h2>
+    <span>
+        The "One-Click" feature of <a href="../../">BSManager</a> is an incredibly convenient tool for Beat Saber users. It allows for easy and quick installation of models for all existing versions of Beat Saber. With this system, you will no longer need to spend hours manually installing models for each specific version of Beat Saber. All you need is one simple click to install models on all versions of Beat Saber you own.
+    </span>
+</div>
+<br>
+<div align="center">
+
+
+
+https://user-images.githubusercontent.com/40648115/213423435-f0fc9dda-1328-449f-ab54-39d28cb6a8ac.mov
```

Comment:
🔴Where is the oneClick downloads for Maps and playlist ?

Reply:
you can click on the images listed in the oneClicks cards (where you enable, disable the oneClick)

Reply:
so amazing, thanks

Reply:
fix